ibm_catalog.json

```diff
@@ -0,0 +1,145 @@
+        "product_kind": "solution",
+        "tags": [
+          "containers",
+          "target_terraform",
+          "terraform",
+          "solution",
+          "support_ibm",
+          "ibm_created",
+          "ibm_beta"
+        ],
+        "keywords": [
+          "openshift",
+          "openshift-ai",
+          "ocp-ai",
+          "cluster",
+          "IaC",
+          "infrastructure as code",
+          "terraform",
+          "solution",
+          "machine learning",
+          "deep learning",
+          "AI",
+          "artificial intelligence"
+        ],
+        "short_description": "Rapidly deploy Red Hat OpenShift clusters pre-configured with AI capabilities for accelerated machine learning workloads.",
+        "long_description": "This automation streamlines the deployment of Red Hat OpenShift cluster with Artifical Intelligence (AI) addon on IBM Cloud. It accelerates the setup of machine learning environments, reducing the complexity of AI infrastructure management and allowing data scientists and developers to focus on model development and deployment. This offering depends on 'Cloud automation for Red Hat OpenShift VPC' to create a base OCP cluster. Please refer to the released version here (link to OCP DA release). Refer [here](https://cloud.ibm.com/docs/containers?topic=containers-supported-cluster-addon-versions#openshift-ai-416) for supported AI addon versions.",
+        "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-ocp-ai/blob/main/README.md",
+        "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-ocp-ai/main/images/rh_ocp_ai.svg",
+        "provider_name": "IBM",
+        "features": [
+          {
+            "title": "AI-Enhanced Red Hat OpenShift Deployment",
+            "description": "Automates the provisioning of a fully configured Red Hat OpenShift on VPC cluster with integrated AI capabilities on IBM Cloud. This includes optimized configurations for machine learning frameworks and simplified deployment of AI-driven applications. Refer the documentation to better understand the service and its capabilities: https://cloud.ibm.com/docs/openshift?topic=openshift-ai-addon-about"
+          }
+        ],
+        "flavors": [
+          {
+            "label": "Openshift AI",
+            "name": "openshift-ai",
+            "install_type": "fullstack",
+            "working_directory": "solutions/ocp-ai",
+            "architecture": {
+              "descriptions": "This architecture deploys a single-zone Red Hat OpenShift on VPC cluster enhanced with an AI addon, accelerating machine learning and AI workload deployments. It outlines the necessary infrastructure components for AI-powered applications and provides a streamlined setup for data science and AI development.",
+              "features": [
+                {
+                  "title": "Create Virtual Private Cloud (VPC)",
+                  "description": "No"
```

Comment:
it is `Yes` as a part of OCP dependency.

Reply:
Done